app-text/csvfix/csvfix-1.6-r1.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit edos2unix toolchain-funcs
+
+DESCRIPTION="A stream editor for manipulating CSV files"
+HOMEPAGE="https://github.com/jheusser/csvfix"
+SRC_URI="https://github.com/jheusser/csvfix/archive/version-${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="dev-libs/expat"
+DEPEND="${RDEPEND}"
+
+S="${WORKDIR}/${PN}-version-${PV}"
```

Comment:
Just a small nit pick. If you have a look into ```skel.ebuild```, S is after SRC_URI :-)

Reply:
All ebuilds that I saw, places S after *DEPEND section.